application.py

```diff
@@ -16,38 +16,45 @@
     url_for,
 )
 from jinja2 import ChainableUndefined
-from http import HTTPStatus
+from dotenv import load_dotenv
+
 from enum import Enum
+from http import HTTPStatus
 
+load_dotenv()
 # Basic logging information
 # logging.basicConfig(level=logging.DEBUG)
-# logger = logging.getLogger(__name__)
-
-# SETTING OF API URL: Change if moving to production
-API_URL = os.getenv("API_URL")
+logger = logging.getLogger(__name__)
 
 # AWS S3 bucket settings
-bucket_name = "keh-tech-audit-tool"
-region_name = os.getenv("AWS_DEFAULT_REGION")
+api_bucket_name = os.getenv("API_BUCKET_NAME")
+region_name = 'eu-west-2'
 s3 = boto3.client("s3", region_name=region_name)
 
-# Standard flask initialisation
-app = Flask(__name__)
-app.secret_key = os.getenv("APP_SECRET_KEY")
-app.jinja_env.undefined = ChainableUndefined
-app.jinja_env.add_extension("jinja2.ext.do")
-
 # GET client keys from S3 bucket using boto3
 def read_client_keys():
+    api_secret_name = os.getenv("API_SECRET_NAME")
+    region_name = "eu-west-2"
+
+    # Create a Secrets Manager client
+    session = boto3.session.Session()
+    client = session.client(
+        service_name='secretsmanager',
+        region_name=region_name
+    )
+
     try:
-        response = s3.get_object(Bucket=bucket_name, Key="client_keys.json")
-        client_keys = json.loads(response["Body"].read().decode("utf-8"))
+        get_secret_value_response = client.get_secret_value(
+            SecretId=api_secret_name
+        )
     except ClientError as e:
-        if e.response["Error"]["Code"] == "NoSuchKey":
-            client_keys = {}
-        else:
-            abort(500, description=f"Error reading client keys: {e}")
-    return client_keys
+        # For a list of exceptions thrown, see
+        # https://docs.aws.amazon.com/secretsmanager/latest/apireference/API_GetSecretValue.html
+        raise e
+
+    secret = get_secret_value_response['SecretString']
```

Comment:
> Will the key `SecretString` in `get_secret_value_response` ever be null or not set? For integrity it might be worth using `.get()` so a value can be gained if it is null [(.get()](https://www.w3schools.com/python/ref_dictionary_get.asp).
> 
> This might happen if there is a secret on SM with the correct name but no key added.

This probably has a very low chance of happening but i'll add it in anyways